app/helpers/assessment_query_helper.rb

```diff
@@ -5,4 +5,88 @@ module AssessmentQueryHelper
   def assessments_by_patient_ids(patient_ids)
     Assessment.where(patient_id: patient_ids).order(:patient_id)
   end
+
+  def search(assessments, search)
+    return assessments if search.blank?
+
+    assessments.where('id like ?', "#{search&.downcase}%").or(
+      assessments.where('who_reported like ?', "#{search&.downcase}%")
+    )
+  end
+
+  def sort(assessments, order, direction)
+    # Order by created_at date by default
+    return assessments.order(created_at: 'desc') if order.blank? || direction.blank?
+
+    # Satisfy brakeman with additional sanitation logic
+    dir = direction == 'asc' ? 'asc' : 'desc'
+
+    case order
+    when 'id'
+      assessments = assessments.order(id: dir)
+    when 'symptomatic'
+      assessments = assessments.order(symptomatic: dir)
+    when 'who_reported'
+      assessments = assessments.order(who_reported: dir)
+    when 'created_at'
+      assessments = assessments.order(created_at: dir)
+    else
```

Comment:
I will be the first to say I do not like this implementation for sorting the symptom columns. It is slow, it requires the use of a new gem, and it's a bit hard to follow. I feel like there _must_ be a better way to handle this sorting and would be happy to brainstorm more, but none of my attempts with a joins/includes worked out in such a way that I could get this order in a Relation by the end.